packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1707,6 +1703,29 @@ export class ContainerRuntime
 		});
 	}
 
+	public getCreateChildSummarizerNodeFn(id: string, createParam: CreateChildSummarizerNodeParam) {
+		return (
+			summarizeInternal: SummarizeInternalFn,
+			getGCDataFn: (fullGC?: boolean) => Promise<IGarbageCollectionData>,
+		) =>
+			this.summarizerNode.createChild(
+				summarizeInternal,
+				id,
+				createParam,
+				undefined,
+				getGCDataFn,
+			);
+	}
+
+	public deleteChildSummarizerNode(id: string) {
```

Comment:
#TODO this can be private. Same as for the DataStoreContext. We shouldn't really be calling this outside the parentContext